README.md

```diff
@@ -598,24 +598,12 @@
 
     ```javascript
     // good
```

Comment:
There are now two `// good` blocks and no bad blocks? was this the intended result? This is kind of confusing if so, maybe they should just have one `// good` comment.